packages/playground/assets/__tests__/assets.spec.ts

```diff
@@ -234,6 +234,12 @@ test('new URL(..., import.meta.url)', async () => {
   expect(await page.textContent('.import-meta-url')).toMatch(assetMatch)
 })
 
+test('new URL(/..., import.meta.url)', async () => {
+  expect(await page.textContent('.import-meta-url-base-path')).toMatch(
+    iconMatch
+  )
+})
+
```

Comment:
On `main`, fixture I added here actually doesn't work for both sets of tests.

During build, if the asset is small enough, it's resolved and optimized into a Data URI despite being in the public folder. Otherwise, it is resolved to be relative the the public base path.

During serve, the import plugin isn't run at all, and this resolves to the root of the site.

Which one of these is more correct? I think ideally, the root path should always resolve from the public base directory. If developers want it to be from the root of the site (as it does with serve mode), they should just omit `import.meta.url`.